readme.md

```diff
@@ -52,6 +52,7 @@ The recommended way to use conjure-java is via a build tool like [gradle-conjure
 * `incubating`: Describes an endpoint as incubating and likely to change. These endpoints are generated with an `@Incubating` annotation.
 * `server-request-context`: Opt into an additional [RequestContext](conjure-undertow-lib/src/main/java/com/palantir/conjure/java/undertow/lib/RequestContext.java) parameter in conjure-undertow interfaces, which allows request metadata to be read, and additional arguments to be associated with the request log.
 * `server-async`: Opt into [asynchronous request processing](#asynchronous-request-processing) in conjure-undertow. The generated interface returns a `ListenableFuture` of the defined return type, allowing processing to occur in the background without blocking the request thread.
+* `deprecated-for-removal`: (applies to Dialogue services only) For endpoint definitions that [are marked deprecated](https://github.com/palantir/conjure/blob/master/docs/spec/conjure_definitions.md#endpointdefinition), indicates that the endpoint will be removed in a future release. This has the effect of Java client being generated with an `@Deprecated(forRemoval = true)` annotation on the endpoint method in the Dialogue service interface.
```

Comment:
> applies to Dialogue services only

This part may not be entirely accurate depending how we handle metrics and such on the server-side. I'd probably strike the parenthetical.

Reply:
ok, removed